src/bot/events/onIssueCommentCreated.ts

```diff
@@ -150,6 +151,12 @@ export const onIssueCommentCreated: WebhookHandler<"issue_comment.created"> = as
 
         const defaultVariables = parsedCommand.configuration.gitlab?.job.variables;
         const overriddenVariables = parsedCommand.variables;
+        const configJobImage = parsedCommand.configuration.gitlab?.job.image;
+        let image: string = typeof configJobImage === "string" ? configJobImage : gitlab.defaultJobImage;
```

Comment:
wait, why do we specify image for all commands in [this PR](https://github.com/paritytech/command-bot-scripts/pull/42/files), while we still have `gitlab.defaultJobImage;` ?

Reply:
Well yea if there's a way to override a local command with -v CMD_IMAGE, and the main image is defined in "GITLAB_JOB_IMAGE" env, there's no need in "image" from .cmd.json 

for CI team there will be 1 place to change it - in our repo, but it will require re-deployment (which is fine I think)
or if they wanna test it - then use CMD_IMAGE var
I am probably closing the PR in scripts https://github.com/paritytech/command-bot-scripts/pull/42 and cleaning up these rows

Reply:
updated 🙏